### PR TITLE
Added Logic to Add New Relativity Version to the Solution Snapshot Da…

### PR DESCRIPTION
**Pull request by aarongilbert123**
…tabase

### CODE REVIEW

File: Chef/AddRelativityVersionToSolutionSnapshotDatabase.ps1 [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    $requestJson = $request | ConvertTo-Json
+    Write-Message "Request Json: $($requestJson)"
+
+    $response = invoke-restmethod -uri $createrelativityversionasyncurl -method post -body $requestjson -contenttype 'application/json' -headers @{"x-csrf-header"="-"}
+    $responsejson = $response | convertto-json
+    Write-Message "Created Relativity Version"
+	Send-Slack-Success-Message $relativityVersion
+  }
+  catch {
+	if($_.ToString().Contains('Relativity Version ' + $relativityVersion + ' already exists')){
```
Comment (calimetikcura) [inline Chef/AddRelativityVersionToSolutionSnapshotDatabase.ps1]: we need a slack message when there is an exception

Comment (aarongilbert123) [reply Chef/AddRelativityVersionToSolutionSnapshotDatabase.ps1]: Done

Comment (aarongilbert123) [reply Chef/AddRelativityVersionToSolutionSnapshotDatabase.ps1]: Done

Comment (TommyBigNose) [reply Chef/AddRelativityVersionToSolutionSnapshotDatabase.ps1]: Shouldn't the name of the function change to Send-Slack-Message (not necessary success)
Also I don't see the 2nd call to the slack function for error. Am I missing something?

**Pull request merged**